src/cli/bootstrap/keys.ts

```diff
@@ -45,3 +45,12 @@ export async function updateAccountKey(account: NameType, newPublicKey: PublicKe
     await eosioContract.updateauth(account.toString(), 'active', 'owner', authority, signer);
     await eosioContract.updateauth(account.toString(), 'owner', 'owner', authority, signer);
 }
+
+export async function updateControllByAccount(account: NameType, owner: string, addCodePermission = false) {
```

Comment:
Probably good to renname the `owner` argument to `controllerAccount`. Otherwise "owner" might be confused with the permission  chain "owner"